datasetinsights/stats/visualization/plots.py

```diff
@@ -127,6 +132,36 @@ def get_color(match):
     colors = [get_color(match) for _, match in match_results]
     return colors
 
+def plot_bboxes3d(image, bboxes, projection=None, colors=None):
+    """ Plot an image with 3D bounding boxes
+    
+    Currently only ground truth images are supported. If a list of colors is not
```

Comment:
What does it mean that only ground truth images are supported?

Reply:
So the other plot bboxes methods have documentation about how the method performs based on if you are using it for 'ground truth' vs. 'prediction.' I'm not sure what would have to change to support prediction, so I wanted to document that. Reworded comment to hopefully clear that up.